IDE/include.am

```diff
@@ -17,5 +17,7 @@ include IDE/VS-ARM/include.am
 include IDE/GCC-ARM/include.am
 include IDE/CSBENCH/include.am
 include IDE/mynewt/include.am
+include IDE/Renesas/cs+/include.am
+include IDE/Renesas/e2studio/include.am
 
-EXTRA_DIST+= IDE/IAR-EWARM IDE/MDK-ARM IDE/MDK5-ARM IDE/MYSQL IDE/LPCXPRESSO IDE/HEXIWEAR
+EXTRA_DIST+= IDE/IAR-EWARM IDE/MDK-ARM IDE/MDK5-ARM IDE/MYSQL IDE/LPCXPRESSO IDE/HEXIWEAR IDE/Renesas/cs+/Projects IDE/Renesas/e2studio/Projects
```

Comment:
There is some typo or case error. `automake: error: cannot open < IDE/Renesas/cs+/include.am: No such file or directory`. Also don't add the dirs to EXTRA_DIST here, since they are already in the cs+ and e2studio include.am.

Reply:
Thank  you for your checking.
I corrected typos and mis-conifgurations.